readme.md

```diff
@@ -78,11 +91,11 @@ For this project, work with your team to build a creative product that you actua
 
 ### Support
 
-We will be responding to [github issues](https://github.com/ga-wdi-exercises/project3/issues).
+The instructors will be responding to [Github issues](https://github.com/ga-wdi-exercises/project3/issues).
 
 > If you'd like a reference for a good question checkout [this lesson](https://github.com/ga-wdi-lessons/effective_questions)
 
-Each group will be given 3 tokens, redeemable at any time for 15 minutes each with an instructor, for a total of 45 minutes per group. No, you can't transfer tokens. And there are no rollover minutes. Just slack or go up to an instructor, they will update the number of tokens assigned to your group recorded in [this file](groups.md).
+Each group will be given 3 tokens, redeemable at any time for 15 minutes each with an instructor of your choice, for a total of 45 minutes per group. No, you can't transfer tokens. And there are no rollover minutes. Just slack the instructor and they will update the number of tokens assigned to your group recorded in [this file](groups.md).
 
```

Comment:
I think "of your choice" is implied since I say it earlier in the paragraph. "the" indicates i'm speaking about the same one.